### PR TITLE
Update Meteor.d.ts

### PR DESCRIPTION
**Pull request by pihentagy**
It's not complete, just those are updates, which I've alreaday ran into.

### CODE REVIEW

**Comment by jankapunkt** [comment]
@pihentagy do you know if prettier is compatible with the d.ts file or do we need to ignore this one?

**Comment by pihentagy** [comment]
@jankapunkt Sorry for that. Fixed prettier.

File: src/Meteor.d.ts [hunk 1]
```diff
@@ -1,33 +1,55 @@
...
   function disconnect(): void;
   function reconnect(): void;
 
-  type Status =
-    | 'change'
-    | 'connected'
-    | 'disconnected'
-    | 'loggingIn'
-    | 'loggingOut';
+  type Status = 'change' | 'connected' | 'disconnected' | 'loggingIn';
```
Comment (jankapunkt) [inline src/Meteor.d.ts:8]: Why did you remove `loggingOut`? It has specifically been added in #108

**Comment by pihentagy** [comment]
Is there anything I need to fix @jankapunkt?

**Comment by pihentagy** [comment]
@jankapunkt ?

**Comment by jankapunkt** [comment]
Sorry @pihentagy missed this in the slack of the recent weeks.

**Review by jankapunkt — APPROVED** [review]

**Comment by pihentagy** [comment]
No worries, thanks for merging

**Comment by pihentagy** [comment]
Won't it trigger a micro version release?

**Comment by jankapunkt** [comment]
https://github.com/meteorrn/meteor-react-native/releases/tag/v2.7.0
